share/site/duckduckgo/base.tx

```diff
@@ -13,42 +13,48 @@
 	<: if !$no_js { include "head_js.tx" } :>
 	<div class="site-wrapper <: if $homepage { :> site-wrapper--home<: } else { :> site-wrapper--static<: } :>  js-site-wrapper">
 	<: if !$no_wrapper { :>
-	  <: if !$no_header { include "header.tx" } :>
-	  <div id="content_wrapper" class="content-wrap">
-	  <: if !$no_content_internal { :>
-		<div id="content" class="content<: if !$no_cw { :>  cw<: } :>">
-			<div id="content_internal" class="content__internal">
-	  <: } :>
-		<: } else {:>
-		  <: if $homepage { :>
+		<: if !$no_header { :>
+			<: if $transparent_header { :>
+				<: include "header.tx" :>
+			<: } else if $flex_header { :>
+				<: include "header_flex.tx" :>
+			<: } else {:>
+				<: include "header.tx" :>
+			<: } :>
+		<: } :>
+		<div id="content_wrapper" class="content-wrap">
```

Comment:
This is just a formatting change right? Let me know if there is anything I should be looking at here ?

Reply:
Yup the only new thing is the check for `$flex_header` -- I may not be able to keep this though as I realized now it breaks the header for the old (currently live) design. I may need to go back to using CSS to create this UI for now.

```perl
<: } else if $flex_header { :>
    <: include "header_flex.tx" :>
```

If you use the 'diff settings' above to hide whitespace changes, the new content becomes much more clear:

<img width="364" alt="duckduckgo-publisher 2018-06-19 10-35-40" src="https://user-images.githubusercontent.com/873785/41604178-9dcd995a-73ac-11e8-8961-30bed1479ca2.png">